src/AppBundle/Entity/JemarcheReport.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * JemarcheReport
+ *
+ * @ORM\Table(name="jemarche_report")
```

Comment:
Can you put the table name in plural form (`jemarche_reports`) to follow the convention?